### ISSUE TITLE
Cannot chain spoks

### ISSUE THREAD

**Issue by bahmutov** [bahmutov/cy-spok#7]
This works (single spok with 2 items)
```js
it('spoks several times', () => {
  const o = {
    one: 1,
    two: 2,
    three: 3,
  }
  cy.wrap(o).should(spok({ one: 1, two: 2 }))
})
```

<img width="521" alt="Screen Shot 2019-12-16 at 4 32 12 PM" src="https://user-images.githubusercontent.com/2212006/70945737-da803180-2023-11ea-97cd-2d4322b43ee5.png">

but adding another assertion there breaks things

```js
it('spoks several times', () => {
  const o = {
    one: 1,
    two: 2,
    three: 3,
  }
  cy.wrap(o)
    .should(spok({ one: 1, two: 2 }))
    .and(spok({ two: 2 }))
})
```

<img width="675" alt="Screen Shot 2019-12-16 at 4 48 03 PM" src="https://user-images.githubusercontent.com/2212006/70945795-ee2b9800-2023-11ea-8ef8-cf74e5693448.png">

**Comment by bahmutov** [comment]
Seems to be caused by https://github.com/cypress-io/cypress/issues/5979

**Comment by sainthkh** [comment]
Fixed in cypress-io/cypress#6128. Now you can chain spoks.

**Comment by bahmutov** [comment]
nice @sainthkh - it is working now, perfect.
